build.sh

```diff
@@ -38,12 +38,12 @@ fi
 
 source .env
 
-if [ -z "$OVERRIDE_DEFAULTS" ]; then
-    echo "reset to default values...";
-    export PROST_RPC_URL="https://rpc-prost1h-proxy.powerloom.io"
-    export PROTOCOL_STATE_CONTRACT="0x10c5E2ee14006B3860d4FdF6B173A30553ea6333"
-    export PROST_CHAIN_ID="11165"
-fi
+# if [ -z "$OVERRIDE_DEFAULTS" ]; then
```

Comment:
Is there any reason to comment this out, @anomit? This part is crucial because it gets hard for us to get snapshotters to update their `.env` files each time a new upgrade is pushed.

Reply:
These values need to be reviewed then. I can restore the commented out section.